src/huggingface_hub/hf_api.py

```diff
@@ -422,8 +424,9 @@ def login(self, username: str, password: str) -> str:
 
         Throws: requests.exceptions.HTTPError if credentials are invalid
         """
-        logging.error(
-            "HfApi.login: This method is deprecated in favor of `set_access_token`."
+        warnings.warn(
+            "HfApi.login: This method is deprecated in favor of `set_access_token`.",
```

Comment:
should we mention when it will be removed? Do we have a deprecation cycle policy?

Reply:
I'm adding a mention to `v0.7`: we should remove this as soon as possible, but there are downstream libraries using it still, so it's important to give them time to switch.

There's no written deprecation cycle policy, but I agree it would be welcome.